src/com/epam/javacourse/homework7/Main.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.javacourse.homework7;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        final String inputFilePath = "src/com/epam/javacourse/resources/poem.txt";
+        String outputFilePath = "src/com/epam/javacourse/resources/poem.txt";
```

Comment:
This one should be final too, and I think you don't need this parameter as it's the same with inputFilePath